### PR TITLE
change method convert from ArrayBuffer to base64

### PR DESCRIPTION
**Pull request by linhnk-ips**
- Fix verify biometric auth

### CODE REVIEW

File: src/pwa/services/webAuth.ts [hunk 1]
```diff
@@ -22,42 +21,44 @@ export const verifyWebAuth = async (): Promise<boolean> => {
...
     };
 
     try {
-        const credential = await navigator.credentials.create({
+        return await navigator.credentials.create({
             publicKey: publicKeyCredentialCreationOptions
-        });
-        registerAuthenticator(credential);
+        }).then(credential => {
+            console.log(credential);
```
Comment (uveta) [inline src/pwa/services/webAuth.ts]: Guess this console log is used only for debugging purposes. It should be removed if so.

**Comment by uveta** [comment]
@linhnk-ips  What is the reason for the change in algorithm? Could you provide more details?

File: src/pwa/services/webAuth.ts (hunk 1 above)
Comment (linhnk-ips) [reply src/pwa/services/webAuth.ts]: thanks. I removed it.

**Comment by linhnk-ips** [comment]
> @linhnk-ips What is the reason for the change in algorithm? Could you provide more details?

@uveta when app register WebAuth, it returns `credential `obj includes `rawId`, which will be used to pass as a parameter when verifying. Returning `rawId `is `ArrayBuffer`, to save it, I convert to `base64`, and when verifying I convert back to `ArrayBuffer`. The method converts it was wrong => verifying is unsuccess (wrong rawId), so I fix it in this pull request.

**Review by uveta — APPROVED** [review]

**Pull request merged**